101-SQL-Injection-Attack-Prevention/nested/microsoft.web/sites.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "name": {
+        "type": "string"
+      },
+      "hostingPlanName": {
+        "type": "string"
+      },
+      "location": {
+        "type": "string"
+      },
+      "hostingEnvironment": {
+        "type": "string",
+        "defaultValue": ""
+      },
+      "serverFarmResourceGroup": {
+        "type": "string",
+        "defaultValue": "[resourceGroup().name]"
+      },
+      "subscriptionId": {
+        "type": "string",
+        "defaultValue": "[subscription().id]"
+      },
+      "tags": {
+        "type": "object"
+      }
+    },
+    "variables": {
+      "tags": {
+        "[concat('hidden-related:', '/subscriptions/', parameters('subscriptionId'),'/resourcegroups/', parameters('serverFarmResourceGroup'), '/providers/Microsoft.Web/serverfarms/', parameters('hostingPlanName'))]": "empty"
+      },
+      "combinedTags": "[union(parameters('tags'),variables('tags'))]"
+    },
+    "resources": [
+      {
+        "apiVersion": "2018-02-01",
+        "name": "[parameters('name')]",
+        "type": "Microsoft.Web/sites",
+        "tags": "[variables('combinedTags')]",
+        "identity": {
+          "type": "SystemAssigned"
```

Comment:
what is this used for?

Reply:
This is used to create and assign an Azure AD principal for the resource